### ISSUE TITLE
Sync profile on restore seed phrase onboarding workflow

### ISSUE THREAD

**Issue by iurimatias** [status-im/status-desktop#5837]
### Description

see figma:
- https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1023%3A298171
- https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=1592%3A128606

Support syncing from desktop, if mobile does not support this yet, then it should be done in a different task.

<img width="868" alt="Screen Shot 2022-05-30 at 7 08 40 PM" src="https://user-images.githubusercontent.com/176720/171066806-6e03d8e8-6f1c-4925-aaef-3e71258b3c89.png">

goals:
- [ ] write feature spec describing syncing in https://github.com/status-im/feature-specs
- [ ] update settings syncing screen
- [ ] Support syncing with another desktop app

note:
- status-go apis for this should already exist, but it's possible some might need to be implemented
- keycard syncing is out of scope
- [PR#5756](https://github.com/status-im/status-desktop/pull/5756) might be relevant if scanning QR codes is required

**Comment by John-44** [comment]
Before any work starts on this issue, the desktop designs need to be updated (they are currently incorrect with regards to a few details).  So check with me before starting work on this
